R/zzz.R

```diff
@@ -0,0 +1,4 @@
+##' @importFrom magrittr %>%
+##' @importFrom rlang .data
+##' @export
+magrittr::`%>%`
```

Comment:
What is going on here?

Reply:
Oh yeah, just copied this from SO - replaced with NULL